core/version.h

```diff
@@ -31,6 +31,7 @@
 #ifndef VERSION_H
 #define VERSION_H
 
+#include "core/typedefs.h"
```

Comment:
Needed for `_MKSTR` used in that file. It just happened to always compile in other files as `typedefs.h` is virtually included everywhere (but in the `.rc` files).